engine/intro.asm

```diff
@@ -231,11 +237,11 @@ IntroPlaceBlackTiles:
 
 IntroMoveMon:
 ; d = number of times to move the mon (2 pixels each time)
-; e: $00 = move Gengar right, $01 = move Gengar left, $ff = move Nidorino right
+; e: 0 = move Gengar right, 1 = move Gengar left, -1 = move Nidorino right
```

Comment:
this is probably where you meant to use the `MOVE_GENGAR_RIGHT` constants

Reply:
you're right, I just updated it